static/src/javascripts/types/ab-tests.d.ts

```diff
@@ -0,0 +1,181 @@
+import type { ReminderFields } from 'common/modules/commercial/templates/acquisitions-epic-reminder';
+import type { TickerSettings } from 'common/modules/commercial/ticker';
```

Comment:
Don’t these need to become relative?

Reply:
the files don't seem to exist, but I don't really want to get into changing the contents in this PR?